test/unit/math/prim/scal/prob/neg_binomial_2_test.cpp

```diff
@@ -202,3 +202,14 @@ TEST(ProbDistributionsNegBinomial, chiSquareGoodnessFitTest4) {
 
   EXPECT_TRUE(chi < quantile(complement(mydist, 1e-6)));
 }
+
```

Comment:
tiny update on this, I have a reasonable test for the derivatives which reveals an error in the derivative with respect to *mu* when phi is large. I'll flag the PR again when it's fixed